flink-yarn/src/main/java/org/apache/flink/yarn/Utils.java

```diff
@@ -465,6 +468,8 @@ static ContainerLaunchContext createTaskExecutorContext(
 			log.debug("Writing TaskManager configuration to {}", taskManagerConfigFile.getAbsolutePath());
 			BootstrapTools.writeConfiguration(taskManagerConfig, taskManagerConfigFile);
 
+			final int replication = yarnConfig.getInt(DFSConfigKeys.DFS_REPLICATION_KEY, DFSConfigKeys.DFS_REPLICATION_DEFAULT);
```

Comment:
Maybe we should use `YarnConfigOptions.FILE_REPLICATION`? The yarn configuration(yarn-site.xml) could not be uploaded from flink client.